Mathlib/Data/List/GetD.lean

```diff
@@ -88,6 +88,11 @@ theorem getD_eq_getD_get? (n : ℕ) : l.getD n d = (l.get? n).getD d := by
   | inl h => rw [getD_eq_getElem _ _ h, get?_eq_get h, get_eq_getElem, Option.getD_some]
   | inr h => rw [getD_eq_default _ _ h, get?_eq_none.mpr h, Option.getD_none]
 
+theorem getD_eq_getD_getElem? (n : ℕ) : l.getD n d = (l[n]?).getD d := by
```

Comment:
This already exists: [`List.getD_eq_getElem?_getD`](https://leanprover-community.github.io/mathlib4_docs/Init/Data/List/Lemmas.html#List.getD_eq_getElem?_getD)